### ISSUE TITLE
Feature request: Server side searching and pagination.

### ISSUE THREAD

**Issue by breaktag** [andrewcourtice/vuetiful#38]
Hi, I love this , however for me to use I would require server side record limiting and pagination support. Basically I have a record list of over 10,000 records with multiple relationships involved as well so currently it takes a while!

**Comment by jayramanuj** [comment]
I would love to have that feature. Facing the same small delay for large number of data.

**Comment by lee-alderdice** [comment]
Yes please, for the exact same reasons as mentioned above.  Basically a replacement to Datatables.io serverside functionality would be a fantastic starting point.

**Comment by IvoPereira** [comment]
+1 on this.

Was evaluating a good DataTable to use and that is the reason why this one is a no.
